src/layers/microscopy-viewer-layer/data-utils/zarr-utils.js

```diff
@@ -1,56 +1,51 @@
-import { slice, openArray } from 'zarr';
-
-async function getData({ arr, channel, x, y, stride, tilingWidth }) {
-  const arrSlice = slice(
-    stride * tilingWidth * y + stride * x,
-    stride * tilingWidth * y + stride * (x + 1)
-  );
-  const dataSlice = await arr.get([arrSlice]);
-  const { data } = dataSlice;
-  const dataObj = {};
-  dataObj[channel] = data;
-  return dataObj;
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Is this necessary?

Reply:
Thanks for pointing this out. Vestigial.